ext/sst/sst_responder_subcomponent.cc

```diff
@@ -205,6 +208,25 @@ SSTResponderSubComponent::portEventHandler(
                 dynamic_cast<SST::Interfaces::StandardMem::Read*>(request)) {
             return;
         }
+        else if (SST::Interfaces::StandardMem::ReadResp* test =
+                dynamic_cast<SST::Interfaces::StandardMem::ReadResp*>(
+                request)) {
+            // functional calls this
+            // gem5::RequestPtr req = std::make_shared<gem5::Request>(
+            //     dynamic_cast<SST::Interfaces::StandardMem::ReadResp*>(
+            //         request)->pAddr,
+            //     dynamic_cast<SST::Interfaces::StandardMem::ReadResp*>(
+            //         request)->size, 0, 0);
+
+            // gem5::PacketPtr pkt = new gem5::Packet(
+            //     req, gem5::MemCmd::ReadResp);
+
+            // // Clear out bus delay notifications
+            // pkt->headerDelay = pkt->payloadDelay = 0;
+            // if (!(responseReceiver->sendTimingResp(pkt)))
+            //     responseQueue.push(pkt);
```

Comment:
Why is all of this commented out code included?

Reply:
this was done to test adding a new memory packet to gem5 on the fly. the issue is that it does not have a destination, so the simulation crashes. i have removed all this code now as it serves no purpose.